wrappers/services.go

```diff
@@ -335,7 +335,27 @@ func StopServices(apps []*snap.AppInfo, reason snap.ServiceStopReason, inter int
 	}
 
 	return nil
+}
+
+// ServicesEnableState builds a map of service names and whether the snap
```

Comment:
Jumping bettwen this and #7437 made me notice that the comment needs a bit of a change, probably something in the lines of:
```
// ServicesEnableState returns a map of service names from the given snap, together with their enable/disable status.
```